initfiles/bin/init_thor

```diff
@@ -29,7 +29,7 @@ rm -f ${SENTINEL}
 killed() {
         echo "Stopping"
         $deploydir/stop_thor $deploydir
-        kill_process ${SENTINEL} ${PID_NAME} 3
+        kill_process ${PID_NAME} thormaster 3 ${SENTINEL}
```

Comment:
I don't think 'thormaster' is enough..
The process name run is a softlink to thormaster_lcr, that looks like e.g. thormaster_mythor

It's normal to have >1 thormaster on the same physical node in some environments.
So grepping for 'thormaster' would not uniquely identify the correct one.